### PR TITLE
RELEASE.md: add instructions for React dep updates

### PR DESCRIPTION
**Pull request by juliusv**
Signed-off-by: Julius Volz <julius.volz@gmail.com>

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -75,6 +67,30 @@ you can skip the dependency update or only update select dependencies. In such a
...
+go mod vendor
+git add go.mod go.sum vendor
+```
+
+#### Updating React dependencies
+
+Either upgrade the dependencies within their existing version constraints as specified in the `package.json` file (see https://docs.npmjs.com/files/package.json#dependencies):
+
+```
+cd web/ui/react-app && yarn upgrade
```
Comment (roidelapluie) [inline RELEASE.md]: Can we know which files to add ?

Comment (juliusv) [reply RELEASE.md]: Done, sorry for the delay.

**Comment by juliusv** [comment]
Rebased on master after conflict.

Does this look ok for merging now?

**Comment by simonpasquier** [comment]
Can you rebase again? Otherwise LGTM.

**Comment by roidelapluie** [comment]
This will be impacted by dependabot

**Comment by gouthamve** [comment]
@juliusv Could you rebase this again :see_no_evil:

**Comment by juliusv** [comment]
@gouthamve Rebased.

**Review by gouthamve — APPROVED** [review]

**Pull request merged**